ykrt/src/compile/jitc_yk/jit_ir/mod.rs

```diff
@@ -504,6 +513,10 @@ impl Module {
     ) -> Result<GuardInfoIdx, CompilationError> {
         GuardInfoIdx::new(self.guard_info.len()).inspect(|_| self.guard_info.push(info))
     }
+
+    pub(crate) fn tilocs(&self) -> &Vec<VarLocation> {
```

Comment:
I expect Clippy will ask you to do `-> &[VarLocation]` ?

Reply:
It didn't but I changed it anyway: [0bbdc78](https://github.com/ykjit/yk/pull/1318/commits/0bbdc78673741da4ccafd26ad5737d207b1c0d87)